### ISSUE TITLE
[$40][MSFT-240] [Register][NVD][Edge] Input default text read-out twice by the screen-reader

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#2895]
**Steps to Reproduce**
1. Go to https://accounts.topcoder.com/member
2. Press Tab button to go through the form

**Expected Result**
Should read-out only one time

**Actual Result**
Input default text read-out twice by the screen-reader

**Device**: Desktop/Labtop
**Operating System**: Windows
**Browser**: Edge

**WCAG Category**: 1. Perceivable
**WCAG Level**: A
**WCAG Success Criterion**: 1.3.1 Info and Relationships
**Screen Reader Used**: Other
**Accessibility Audit Tool used**: N/A

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=58da9b68%2Df2a3%2D45d2%2Dbbae%2D15abdfd0865b&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&sortField=Modified&isAscending=false&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2Fcodejam%2DAccessibility%2F74

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096237 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096237 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by Nachiappa** [comment]
I tried to reproduce this issue , but couldn't . Here is my recording 


[Topcoder - Google Chrome 2019-07-23 08-20-22.zip](https://github.com/topcoder-platform/community-app/files/3419984/Topcoder.-.Google.Chrome.2019-07-23.08-20-22.zip)

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096237 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30097858 has been created for this ticket.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30097858 has been updated - it has been assigned to rashmi73.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30097858 has been updated - it has been assigned to rohitgupta_.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by r0hit-gupta** [comment]
@veshu @codeMinter unable to reproduce. Should have been fixed by other PRs probably.

Test URL - https://accounts.topcoder.com/member
Env - Windows/Edge
Screen Reader Used - NVDA

**Comment by drasticdpk** [comment]
@r0hit-gupta , 
Here is the reference link :
https://monosnap.com/file/1G5ecdK5ZSrguvdS8vhtNLdtVt9brS 
FYI, we are not having aria-label in password(and other control in registration too) but have at user name. Can we remove aria-label from username ? Please confirm.
![image](https://user-images.githubusercontent.com/25297859/62192349-2bed9400-b395-11e9-9e9a-2853c224b63b.png)
![image](https://user-images.githubusercontent.com/25297859/62192620-aae2cc80-b395-11e9-9a2f-3315c40d1dda.png)

**Comment by r0hit-gupta** [comment]
@drasticdpk will look again deeper into the issue.

**Comment by r0hit-gupta** [comment]
@drasticdpk @nkumar-topcoder there is no `aria-label` on the username field for branch `qa-accessibility`. https://github.com/appirio-tech/accounts-app/blob/58e0f861f980ad14a1ab0e78ee22d14bad62b91a/app/views/tc/login.jade#L24

Can you please check the deployment? Thanks!

**Comment by nithyaasworld** [comment]
@r0hit-gupta Its not repeating the default text. But it reads out all the validation error message associated with that field (for both username and password fields).

Could you check please?

**Comment by r0hit-gupta** [comment]
Looking into it.

**Comment by veshu** [comment]
@r0hit-gupta Reminder on this.

**Comment by r0hit-gupta** [comment]
@veshu sorry for the delay. @drasticdpk @nithyaasworld please review.

**Comment by mishacucicea** [comment]
@nkumar-topcoder Was this deployed on dev environment?

**Comment by nkumar-topcoder** [comment]
> @nkumar-topcoder Was this deployed on dev environment?

@mishacucicea @tosha5252 @drasticdpk this deployed to `qa-accesibility` branch ideally you should be able to test in DEV, if not let me know i will merge to `dev` branch and you can re-test

**Comment by tosha5252** [comment]
Announcement of fields twice is still happening 

Win10/Edge/NVDA

**Comment by nkumar-topcoder** [comment]
@r0hit-gupta can you please check comments from tosha5252 also pls verify code your fix is there at`qa-accessibility`. I have not merged into DEV yet.

**Comment by r0hit-gupta** [comment]
@nkumar-topcoder will do that.

**Comment by r0hit-gupta** [comment]
@nkumar-topcoder @tosha5252 yes my fix is present in `qa-accessibility` branch but not in `dev` branch. How to proceed?

**Comment by nkumar-topcoder** [comment]
> Announcement of fields twice is still happening
> 
> Win10/Edge/NVDA

@r0hit-gupta I am referring the above comment from tosha. If `Announcement of fields happening twice` and this is `qa-accessibility` branch then it's an issue. Hence, I didn't merge to DEV. Can you please check.

**Comment by tosha5252** [comment]
@nkumar-topcoder @r0hit-gupta Once it is merged to dev, then it can be re-tested.

**Comment by r0hit-gupta** [comment]
@nkumar-topcoder @tosha5252 everything is working fine for me locally on branch `qa-accessibility`. 

Win/Edge/NVDA

**Comment by nkumar-topcoder** [comment]
@tosha5252 I have merged to dev. can you pls check.

**Comment by tosha5252** [comment]
Announcement of fields twice is still happening
Win10/Edge/NVDA

Pls see video
[TinyTake by MangoApps-19-08-2019-10-57-56.zip](https://github.com/topcoder-platform/community-app/files/3518669/TinyTake.by.MangoApps-19-08-2019-10-57-56.zip)

**Comment by nkumar-topcoder** [comment]
@r0hit-gupta can you please check. Earlier too it had issue on `qa-accessibility` branch post merge to `dev` same status.

**Comment by r0hit-gupta** [comment]
@nkumar-topcoder @tosha5252 I am really sorry but I had fixed only the login page. The issue body mentioned the URL https://accounts.topcoder.com/member so I did not change the registration form. Will fix it now. @tosha5252 thanks for the video reference.

**Comment by nkumar-topcoder** [comment]
> @nkumar-topcoder @tosha5252 I am really sorry but I had fixed only the login page. The issue body mentioned the URL https://accounts.topcoder.com/member so I did not change the registration form. Will fix it now. @tosha5252 thanks for the video reference.

thanks @tosha5252

**Comment by r0hit-gupta** [comment]
@nkumar-topcoder @tosha5252 @mishacucicea created a new PR. This should properly fix both the registration page and the login page. Please review.

**Comment by crazyk07** [comment]
Closing ticket due to shift in priority, will not be able to complete but will still pay for effort rendered.

**Comment by crazyk07** [comment]
Payment task has been updated: https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30097858<br/><br/>```This is an automated message for crazyk via Topcoder X```
